presto-main/src/main/java/com/facebook/presto/util/CompilerUtils.java

```diff
@@ -37,6 +40,9 @@
     private static final AtomicLong CLASS_ID = new AtomicLong();
     private static final DateTimeFormatter TIMESTAMP_FORMAT = DateTimeFormatter.ofPattern("YYYYMMdd_HHmmss");
 
+    public static final String ERROR_LARGE_BYTECODE = "Query results in large bytecode exceeding the limits imposed by JVM";
+
+
```

Comment:
remove extra newline. This won't pass style check.

Reply:
done